samples/client/petstore/elm/src/Request/Pet.elm

```diff
@@ -20,6 +20,25 @@ import Json.Decode as Decode
 import Url.Builder as Url
 
 
+type Status
```

Comment:
@eriktim for Elm, is there any naming convention for enum? 

In other languages, we may add a prefix/suffix (e.g. StatusEnum). This can help avoid naming conflicts with model names.

Reply:
Thanks @wing328 
You have point here but custom types (as these would be called in Elm) are so common I'd rather not suffix them with `Enum`. Your remark does remind me I should actually start using `Data.Status` for all imported types to avoid conflicts. I'll do that in another PR.

Reply:
👌